tests/conftest.py

```diff
@@ -0,0 +1,190 @@
+        from garden_ai.app.main import app as cli_app
+
+        yield cli_app
+
+
+@pytest.fixture
+def mock_authorizer_tuple(mocker):
+    """Return mocks of the Globus authorizer for the GardenClient."""
+    mock_authorizer = mocker.Mock()
+    mock_authorizer_constructor = mocker.patch(
+        "garden_ai.client.RefreshTokenAuthorizer", return_value=mock_authorizer
+    )
+    return mock_authorizer_constructor, mock_authorizer
+
+
+@pytest.fixture
+def identity_jwt() -> str:
+    """Return a mock Globus Auth JWT"""
+    return "eyJhbGciOiJSUzUxMiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiI2YzllMjIzZi1jMjE1LTRjMjYtOWFiYi0yNjJkYmNlMDAwMWMiLCJlbWFpbCI6IndpbGxlbmdsZXJAdWNoaWNhZ28uZWR1IiwibGFzdF9hdXRoZW50aWNhdGlvbiI6MTY3NjU2MjEyNiwiaWRlbnRpdHlfc2V0IjpbeyJzdWIiOiI2YzllMjIzZi1jMjE1LTRjMjYtOWFiYi0yNjJkYmNlMDAwMWMiLCJlbWFpbCI6IndpbGxlbmdsZXJAdWNoaWNhZ28uZWR1IiwibGFzdF9hdXRoZW50aWNhdGlvbiI6MTY3NjU2MjEyNn1dLCJpc3MiOiJodHRwczovL2F1dGguZ2xvYnVzLm9yZyIsImF1ZCI6ImNmOWY4OTM4LWZiNzItNDM5Yy1hNzBiLTg1YWRkZjFiODUzOSIsImV4cCI6MTY3Nzc4NzMzNCwiaWF0IjoxNjc3NjE0NTM0LCJhdF9oYXNoIjoiT1VMX0s3ZmVyNXNBdk03cEI0dXJnNE95dXZVOUxEcGh3SnhCX1VUXzY1cyJ9.TssuMsFeKFQH9Bd29c2Bj0V_f-KkN8alGtHnZOZbg5AQwVPokthHRA4bro7IHuOCXIoh3kX03KPNcLfyRRM5JN1d4SKl0L9KpkJB45BkKzKcg8KPgChOzs_9jRiiDwmXvIpgWiUNVI4grHIEYVpB_VdFdKw6EwWQgu6ZrN_2rvpa45Pc-NZ_-WR4WDFAx2Hak7sXRXslY_1ftlRgV9348uwp78jh1pnXft-mpgpzwqHVALLKgzecCESYmaipWTd3-atczpH9SPIxOn7DoiX2I2Nhn_8IkrhIZnbmtOyY7wINrSGFonN49AncVTNq9AfIngZB26spUByHW4mLB6E2Mw"  # noqa: E501
+
+
+@pytest.fixture
+def token() -> dict:
+    """Return a dict with refresh_token, access_token, and expires_at_seconds fields."""
+    return {
+        "refresh_token": "MyRefreshToken",
+        "access_token": "MyAccessToken",
+        "expires_at_seconds": 1024,
+    }
+
+
+@pytest.fixture
+def mock_keystore(mocker):
+    """Mock the GardenFileAdapter in to have a ketstore for tests"""
+    mock_keystore = mocker.MagicMock(GardenFileAdapter)
+    mocker.patch("garden_ai.client.GardenFileAdapter").return_value = mock_keystore
+    return mock_keystore
+
+
+@pytest.fixture
+def patch_backend_client_requests(mocker, garden_nested_metadata_json) -> None:
+    """Patches BackendClient methods that make network requests."""
+    backend_client = "garden_ai.backend_client.BackendClient"
+    mocker.patch(
+        f"{backend_client}.get_user_info",
+        return_value={"email": "fake@email.com"},
+    )
+
+    mocker.patch(
+        f"{backend_client}.put_garden",
+        return_value=Garden._from_nested_metadata(garden_nested_metadata_json),
+    )
+
+
+@pytest.fixture
+def patch_garden_constants(mocker, tmp_path):
+    """Patches fields in GardenConstants with temp values for tests."""
+    with mocker.patch.object(GardenConstants, "GARDEN_DIR", tmp_path):
+        with mocker.patch.object(
+            GardenConstants, "GARDEN_KEY_STORE", tmp_path / "tokens.json"
+        ):
+            yield
+
+
+@pytest.fixture
+def garden_client(
```

Comment:
I thought about doing that, my thinking was, as I move to integration tests, it will be nice to be able to use the same client fixture and have separate control of the backend client.